### ISSUE TITLE
The cut on the spectrogram is not perfect

### ISSUE THREAD

**Issue by wocmultimedia** [SubtitleEdit/subtitleedit#7267]
Hi,
After a while I'm using your great software but I found a problem. When I make a cut on the waveform it doesn't make it correct, but is always ahead and/or  split the subtitle lines in another position. See shot.
![image](https://github.com/SubtitleEdit/subtitleedit/assets/2352240/c23300c2-a5b7-486e-bb91-6dc2093256fc)

The cut I made was where the word "era" finish but the cut added some extra word "Este nos indica". I checked the options but there's no adjustment for this cutting. If it exists I didn't it.
Thanks,
Angelo

**Comment by niksedk** [comment]
How exactly did you split? SE offers many different ways of doing a split.

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/dc167d4b-3323-4138-a468-f2af7c0e261e)

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/46d825ac-764c-4e86-9b3f-581e462ef11e)

Also, adding the text or attaching the subtitle makes it easier for me to re-create

**Comment by wocmultimedia** [comment]
Hi Nik,
Thank you for your quick reply. I use this shortcut and also I tried the third one shown in the list adding a custom shortcut. But the problem is that in both cases it never cut at video position but always forward to the right. 
Here's a file zip with a link to download the video and the text, so you can recreate it.
[material_to_make_testzip.zip](https://github.com/SubtitleEdit/subtitleedit/files/12393071/material_to_make_testzip.zip)

Let me know if  do you need something else.
Thanks
Angelo

**Comment by niksedk** [comment]
I works perfectly fine here (video offset 10 hours, using mpv as video player)

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/6a963033-bdf0-46df-b83a-18ea0f53f17a)

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/a2f12b43-fef3-4a38-9b1b-d095bdd1b410)

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/ff49ed6d-ce34-454a-884e-429e39a33ace)

**Comment by wocmultimedia** [comment]
What do you mean?
Do I have to download another player not using the one appearing when I open the software?
What do you mean for offset 10 hours? can you explain it, what is wrong with my settings?
Thanks

22 ago 2023 18:40:06 Nikolaj Olsson ***@***.***>:

**Comment by wocmultimedia** [comment]
The cut on the waveform do not cut the title text in the position of the wave, and when I cut on the window text it cuts ahead of the point I chose. I used the Prime settings at 25fps

22 ago 2023 18:40:06 Nikolaj Olsson ***@***.***>:

**Comment by wocmultimedia** [comment]
Hi Nik,
it is so difficult to tell me what I have to do about the player you have mentioned? 
Do I have to download a new player?

**Comment by niksedk** [comment]
You can try with mpv player... go to Options - Settings - Video player, click on `Download mpv lib`

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/30e7ac0e-e2c4-4e22-af99-0a40e50da00d)

**Comment by niksedk** [comment]
You have to set the video position in the waveform, then click in the text box where you want to split the text

**Comment by wocmultimedia** [comment]
> You have to set the video position in the waveform, then click in the text box where you want to split the text

This is what I do but with my situation it goes ahead, I will make a new try and report it to you. Thanks

**Comment by wocmultimedia** [comment]
I Nik,
I reinstalled the software and the mpv library. The problem is simple the waveform representation is not linked to text in the text box. I mean that if I set the video cursor in a point and right click on the waveform then use Split it cuts correctly but the text in the box is not corresponding to text in the video position cut.
The same if I go to text box and select a point it cuts ahead in the spectrogram.
The only way is to do as you told me setting the video cursor and then go to text box and choose the point of the text then click split at video/cursor point.
That's not very quick, it should be easier having a point set in the spectrogram and split directly the text without have to rego to text box.
Anyhow thanks a lot
